app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,18 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
+
+    }
+
+    override fun anotherMethod() {
+        var some = 1
```

Comment:
Remove this unused "some" local variable. [Details](https://fb41-221-141-140-219.jp.ngrok.io/project/issues?id=musooff_Topik_AYB-ZFlS8-xhCYxQvxYf&issues=AYCHiV2p0n16_HoDUs-L&open=AYCHiV2p0n16_HoDUs-L)